charts/fluid/fluid/crds/data.fluid.io_dataloads.yaml

```diff
@@ -1,143 +1,146 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
`apiextensions.k8s.io/v1` is supported since v1.16, what about k8s v1.14/v1.15 ?